test/serializers/associations_test.rb

```diff
@@ -145,6 +145,24 @@ def test_associations_custom_keys
         assert expected_association_keys.include? :writer
         assert expected_association_keys.include? :site
       end
+
+      def test_belongs_to_dosnt_load_record
+        post = Post.new({ id: 997 })
+
+        def post.blog_id
+          996
+        end
+        def post.blog
+          fail "should use blog_id"
+        end
+
+        post_serializer = Class.new(ActiveModel::Serializer) do
+          belongs_to :blog, embed: :ids
+        end
+
+        adapter = ActiveModel::Serializer::Adapter::JsonApi.new(post_serializer.new(post))
```

Comment:
almost worked.

i took the code from [here](https://github.com/rails-api/active_model_serializers/blob/master/test/serializers/attribute_test.rb#L54-L55)